bridge/reports/templates/reports/UnsafeContent.html

```diff
@@ -41,6 +41,16 @@
                             <i class="ui sitemap icon"></i>
                         </button>
                     </div>
+                    <div class="ui buttons icon tiny orange">
```

Comment:
Button should be green here

Reply:
Could you attach an image. What it is a button?

Reply:
![image](https://github.com/mutilin/klever/assets/22963832/c696988a-241e-460b-a0dc-d80408998f19)
2 new butoons are orange and added in different sections

Reply:
I agree, that the section should be one for two new buttons, but may be, different from old ones. But I am not sure, that the color should be the same.
Green buttons expand the trace? And new one is for search.